src/test/java/api/support/builders/LoanPolicyBuilder.java

```diff
@@ -322,6 +322,31 @@ public LoanPolicyBuilder withLoanable(boolean loanable) {
       this.gracePeriod);
   }
 
+  public LoanPolicyBuilder withLoanPeriod(JsonObject loanPeriod) {
```

Comment:
Why is the existing builder method `rolling(Period period)` not appropriate for defining the rolling loan period?

Reply:
It's perfectly appropriate, thanks for pointing that out.